model/Core/Vocabularies/HashAlgorithm.md

```diff
@@ -37,4 +40,5 @@ and is a one-way function, that is, a function which is practically infeasible t
 - sha512: TODOdescription
 - spdxPvcSha1: TODOdescription
 - spdxPvcSha256: TODOdescription
+- sphincs+: TODOdescription
```

Comment:
not sure, if `+` is a valid character here. I am also sure, whom to ask or how to find out. But at least translated to enums, this would cause problems.

Reply:
yeah,  I was worried about that.  ok, will change it to sphincsPlus

Reply:
@maxhbr - fixed in file,  please re-review